srcs/init/variable.c

```diff
@@ -0,0 +1,71 @@
+#include "shell_env.h"
+#include "utils.h"
+
+t_variable *create_variable(char *name, char *value, bool exported)
+{
+	t_variable *v;
+
+	v = (t_variable *)malloc_or_die(sizeof(t_variable));
```

Comment:
+ C-Pabiendcastermalloc: https://stackoverflow.com/questions/605845/do-i-cast-the-result-of-malloc#605858
+ sizeof(*v) aulieu de sizeof(t_variable) nan ? (ya une explication pour ça dans le post aussi)